examples/steal_split/Move.toml

```diff
@@ -0,0 +1,14 @@
+[package]
+name = "examples/steal_split"
+version = "0.0.1"
+
+[dependencies]
+MoveStdlib = { git = "https://github.com/rooch-network/rooch.git", subdir = "moveos/moveos-stdlib/move-stdlib", rev = "main" }
+MoveosStdlib = { git = "https://github.com/rooch-network/rooch.git", subdir = "moveos/moveos-stdlib/moveos-stdlib", rev = "main" }
+RoochFramework = { git = "https://github.com/rooch-network/rooch.git", subdir = "crates/rooch-framework", rev = "main" }
+
+[addresses]
+steal_split =  "0x1b1272eece1923fa84926d5c54ed2b3bd00d55713ba41c6d2d398c3cfada4b2"
```

Comment:
Use `steal_split = "_"` for convenient user deployment

Reply:
Thanks, It has been updated now.